content/docs/README.md

```diff
@@ -58,6 +58,5 @@ Here is an example of setting up a new cluster:
 kind create cluster
 export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
 kubectl delete storageclass standard
```

Comment:
Oops, this one is redundant too :)

```suggestion
```

Reply:
`local-path-storage.yaml` installs `standard` storageClass as well, don't know why.

I guess deleting or re-creating `standard` storageClass can be left alone (without it being existed or not)

Reply:
Yeah, the `local-path-storage.yaml` creates the `standard` storage class (if it doesn't exist), but it forcibly sets the default annotation to `false` so that it is not actually used. That way you can configure the kind cluster with just a single `kubectl apply`.